packages/lodestar-beacon-state-transition/src/util/seed.ts

```diff
@@ -26,7 +25,7 @@ import {DomainType} from "../constants";
  */
 export function computeShuffledIndex(
   config: IBeaconConfig,
-  index: ValidatorIndex,
```

Comment:
@mpetrunic the meaning of this is not validator index so we have to make it as `number` to avoid the confusion, that's what the spec PR asked for.